src/components/newsletter/newsletter-sign-up.tsx

```diff
@@ -0,0 +1,264 @@
+    text-align: left;
+    margin-top: 1.375rem;
+  }
+
+  a {
+    text-decoration: underline;
+    font-weight: 600;
+  }
+
+  input[type="text"].error,
+  input[type="email"].error {
+    border-color: var(--error);
+  }
+
+  .input-error {
+    color: var(--error);
+    line-height: 1.625rem;
+    margin-top: 0.75rem;
+    display: none;
+  }
+
+  input[type="text"].error + .input-error,
+  input[type="email"].error + .input-error {
+    display: block;
+  }
+
+  p.response {
+    display: none;
+    margin-bottom: 1rem;
+  }
+
+  p.response.success {
+    display: block;
+    color: var(--midnight);
+  }
+
+  p.response.error {
+    display: block;
+    color: var(--error);
+  }
+
+  .submitSection {
+    label {
+      width: 1px;
+    }
+  }
+`;
+
+type SignUpStatus = {
+  success: boolean;
+  msg: string;
+};
+
+const NewsletterSignUp = () => {
+  const [email, setEmail] = useState(undefined);
+  const [emailError, setEmailError] = useState(false);
+  const [name, setName] = useState(undefined);
+  const [nameError, setNameError] = useState(false);
+  const [mcResponse, setMcResponse] = useState<SignUpStatus | undefined>(
+    undefined
+  );
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+
+    setMcResponse(undefined);
+
+    if (!name) setNameError(true);
+    if (!email) setEmailError(true);
+    if (!name || !email) return;
+
+    const response = await addToMailchimp(email, { FNAME: name });
+
+    handleResult(response.result, response.msg);
+  };
+
+  const handleResult = (result: string, message: string) => {
+    const success = result === "success";
+    if (!success) {
+      const msg = message.includes("already subscribed")
+        ? "This email has already been subscribed."
+        : "Something went wrong, please try again later.";
+      setMcResponse({ success, msg });
+    } else {
+      setMcResponse({ success, msg: "Thank you for subscribing!" });
+    }
+  };
+
+  return (
+    <>
+      <section css={newsletterSignUpStyles}>
+        <header>
+          <h2>Join our mission</h2>
+          <p>
+            We’ve got bold ambitions and need impact focused leaders to join us
+            on our journey. Sign up to our newsletter to be the first to hear
+            about our events and stay up to date on our future plans.
+          </p>
+        </header>
+
+        <article>
+          <form onSubmit={handleSubmit}>
+            <fieldset>
+              <section>
+                <label htmlFor="fullNameInput">Full name</label>
+                <input
+                  type="text"
+                  name="fullName"
+                  id="fullNameInput"
+                  className={nameError ? "error" : ""}
+                  onChange={e => {
+                    setNameError(false);
+                    setName(e.target.value);
+                  }}
+                />
+                <p className="input-error">Please enter your name.</p>
+              </section>
```

Comment:
since we already have two of these inputs with very little difference between them, it might be worth making a reuseable component for this if we expect a third anytime soon